<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorerView</class>
 <widget class="QDialog" name="ScorerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>719</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="19">
    <widget class="QLabel" name="SlineThree">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="16" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Legs&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt; :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line_8">
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="16">
    <widget class="QPushButton" name="ValadationYes">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
   <item row="8" column="19">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Next Leg</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Current Throw :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CurrentThrowLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Last Throw :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LastThrowLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="19" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="ValadationNo">
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="3">
    <widget class="Line" name="line_4">
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="15" rowspan="2">
    <widget class="Line" name="line_5">
     <property name="baseSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Match Settings :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13" rowspan="2">
    <widget class="Line" name="line_11">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="14" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Validation: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QLabel" name="SlingOne">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="19">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Finish Match</string>
     </property>
    </widget>
   </item>
   <item row="3" column="16" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="Line" name="line_3">
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="13" colspan="7">
    <widget class="Line" name="line_6">
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="2">
       <widget class="QCheckBox" name="MatchStats">
        <property name="text">
         <string>Match</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="PlayerTwoStats">
        <property name="text">
         <string>P2 Stats</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QCheckBox" name="CurrentPlayerStats">
        <property name="text">
         <string>Current Player</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QCheckBox" name="WinPercentages">
        <property name="text">
         <string>Wins%</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="PersonalStats">
        <property name="text">
         <string>Pesonal Stats</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QCheckBox" name="RankedStats">
        <property name="text">
         <string>Ranked</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Display : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QCheckBox" name="NumberOf180s">
        <property name="text">
         <string>180's</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="PlayerOneAndPlayerTwoStats">
        <property name="text">
         <string>Both Stats</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="PlayerOneStats">
        <property name="text">
         <string>P1 Stats</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2" rowspan="3" colspan="18">
    <widget class="Line" name="line_2">
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="18" rowspan="2">
    <widget class="Line" name="line_9">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="17">
    <widget class="QLabel" name="LegNumber">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="QLabel" name="SlingTwo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="13">
    <widget class="Widget" name="widget" native="true">
     <property name="baseSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>line_2</zorder>
  <zorder>line_8</zorder>
  <zorder>line_6</zorder>
  <zorder>label_3</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>ValadationNo</zorder>
  <zorder>ValadationYes</zorder>
  <zorder>tableWidget</zorder>
  <zorder>line_11</zorder>
  <zorder>line_5</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>label_6</zorder>
  <zorder>SlingOne</zorder>
  <zorder>SlineThree</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>line_9</zorder>
  <zorder>LegNumber</zorder>
  <zorder>SlingTwo</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ValadationNo</sender>
   <signal>clicked()</signal>
   <receiver>SlingOne</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1222</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>1079</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ValadationNo</sender>
   <signal>clicked()</signal>
   <receiver>SlingTwo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1220</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>1159</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ValadationNo</sender>
   <signal>clicked()</signal>
   <receiver>SlineThree</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1229</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>1219</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
